src/trel_dnssd/trel_dnssd.cpp

```diff
@@ -478,14 +478,9 @@ void TrelDnssd::Peer::ReadExtAddrFromTxtData(void)
     {
         if (StringUtils::EqualCaseInsensitive(txtEntry.mName, kTxtRecordExtAddressKey))
         {
-            char extAddrHexBuf[sizeof(mExtAddr) * 2 + 1];
+            VerifyOrExit(txtEntry.mValue.size() == sizeof(mExtAddr));
```

Comment:
should we add a debug level log here?

Reply:
Make sense. Will add.

Reply:
Added a `INFO` log if we can't dissect Extended Address from TXT data. 
BTW, this error should not happen in practise.